# 1.4 异常捕获
Java用`throws`在方法体前声明受检异常, 用`throw`在方法体中抛出异常, 用`try-catch-finally`在调用处捕获异常。

*注*: 方法复写时, 同返回值可以返回父类声明的相容类型一样，子类也可以声明与抛出父类声明的异常的相容类型。


![Test](./.asset/catchs.png)

